src/main/kotlin/io/arrowkt/repository/interpreter/AccountRepository.kt

```diff
@@ -11,7 +11,7 @@ import io.arrowkt.repository.AccountRepository
 import io.arrowkt.today
 import java.time.LocalDate
 
-object AccountInMemoryRepository : AccountRepository {
+class AccountInMemoryRepository : AccountRepository {
```

Comment:
Why the change?

Reply:
Oh, right, state :D

Reply:
yup, each use case in the main app creates a new repository